### ISSUE TITLE
[BUG] [Blood] Load failure - segmentation fault

### ISSUE THREAD

**Issue by MavisCelus19201** [ZDoom/Raze#1078]
### Raze version

1.9.1

### Which game are you running with Raze?

Blood

### What Operating System are you using?

Linux x86_64

### Please describe your specific OS version

Manjaro Linux 6.6.19-1

### Relevant hardware info

Intel CPU

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

Starting when I was testing 1.9.1, anytime I try to load up Blood, I just get a gamecrash regardless of my setup, even using a fresh ini and messing around with the Blood directory, checking files and making sure things were in place.

Trying it with Windowed or Fullscreen Mode, OpenGL or Vulkan, with a fresh .ini generated all produce the same error regardless of what I try.

I've only run into this issue with Blood, testing Shadow Warrior and Duke Nukem 3D did not show this behavior.

I tested the standalone 1.9.1 release and the flatpak, both had the exact same issue.

### Steps to reproduce the behaviour.

Explain how to reproduce
1. Open up Raze, with Blood recognized in its IWAD path.
2. Try to open the game.
3.

### Your configuration

```ini
raze.ini = https://cdn.discordapp.com/attachments/494612899214524428/1222751391462985818/raze.ini?ex=66175af9&is=6604e5f9&hm=228430c92d268fa960ac8812db097a38c64542abcf375e6142fd22e7d39c4e3b&
```

### Provide a Log

*** Fatal Error ***
!!! Failed to exec debug process
[raze-crash.log](https://github.com/ZDoom/Raze/files/14783568/raze-crash.log)

**Comment by coelckers** [comment]
I cannot do anything with this crash log. It has no debug symbols so there's no way to find out where the crash happened.

**Comment by Fil0sOFF** [comment]
I have the same SegFault with 1.9.1 on default configuration, but on Debian sid.
Log with debug symbols is attached.
[raze-crash.log](https://github.com/ZDoom/Raze/files/14903374/raze-crash.log)

I tried to build from master to check if it's already fixed, but was unable to do so due to some `internal compiler error`.

**Comment by coelckers** [comment]
Should be fixed in 5782206a9138003fe0b9b80ce95012fd1f09efab although this should have been caught before . no idea why it didn't,,,
